### PR TITLE
Explain why application link must be provided, close #358

### LINKED ISSUE #358: What to do when self-organizing selection & application, but Diversity Tickets entry _is_ the application form
We are handling both the selection and application process ourselves, and in the application details write:

> Then, send a mail to events@nextcloud.com with subject [Diversity ticket] YOUR NAME and a link to your contribution, whatever it is!

We wouldn’t like to use a Google form for this, so I wonder if it’s possible to accept email addresses for submissions too? Or if in this case the application details are enough? Putting [a mailto-link in the application details](https://github.com/nextcloud/server/wiki/Diversity-Tickets#application-details) results in an error "Application link is invalid".

Because if we put a section about that on the website it defeats the purpose a bit since we would like to link to Diversity Tickets there. (Sorry in advance because it might be an edge-case. It would be good to know what the best practice here is. :)

### PR DESCRIPTION
**Pull request by lislis**
I added a section in the FAQs about this as discussed in ticket #358 and a link from the corresponding part in the submit event form to the FAQ.

Screenshot from 2019-04-18 16-41-25
Screenshot from 2019-04-18 16-42-43

What is missing is:
- [ ] prove read and agree on the copy (ping @alicetragedy )
- [ ] translate ES
- [ ] translate FR

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -462,6 +462,9 @@ en:
       how_can_i_submit_my_event_answer: "<p>To submit your event, click on \"Submit Event\" and fill out all the details. Your personal information (name and email) will not be public and is only asked so that we can contact you.</p>"
       how_can_i_add_tags: "How can I add a new tag to my event?"
       how_can_i_add_tags_answer: "<p>We currently only allow site administrators to add new tags; if you'd like to tag your event with a tag that is currently not available from the list, please drop us a line at %{travis_email}. We will try to process your request as quickly as possible and add your chosen tag to your event submission.</p>"
+      why_provide_a_link: "Why do I have to provide a link when I manage application and selection myself?"
```
Comment (alicetragedy) [inline config/locales/en.yml]: Would add a “the” here, meaning: “Why do I have to provide a link when I manage the application and selection myself?”

File: config/locales/en.yml [hunk 2]
```diff
@@ -462,6 +462,9 @@ en:
...
+      why_provide_a_link_short: "Why do I have to provide a link?"
+      why_provide_a_link_answer: "When you manage the application and selection process yourself it is important to show applicants (and your community) that you are serious about diversity and inclusion and you are trying to make an effort. A link to a form for the application or a dedicated page on your event's page explaining how you will choose from applicants and what you're looking for and why and what you want to achieve by offering diversity tickets goes a long way."
```
Comment (alicetragedy) [inline config/locales/en.yml]: I think this is a great answer but a bit lengthy so I would change it (slightly) to make it easier to read:
“When you manage the application and selection process yourself it's important to show applicants (and your community) that you are serious about diversity and inclusion and you are trying to make an effort. A link to a form for the application or a dedicated page on your event's page explaining how you will choose from applicants, what you're looking for and why, and what you want to achieve by offering diversity tickets goes a long way.”

Comment (alicetragedy) [reply config/locales/en.yml]: (I pretty much only added commas and used a contraction "it is" => "it's").

**Comment by lislis** [comment]
> I can ping the two people who added the translations to see if they can translate the additional strings 

That would be great @alicetragedy! Would you like to wait with merging until there are translations or merge this and add them in later PRs?

I would wait with deploying until everything is ready of course :)

**Comment by alicetragedy** [comment]
@lislis we can def merge this and add the translations in separate PRs. :)

**Comment by lislis** [comment]
Cool beans, then I'll write two tickets for the missing translations.

**Pull request merged**